web-registry/src/types/ledger/ecocredit.ts

```diff
@@ -12,6 +11,15 @@ export interface BatchInfoWithSupply extends BatchInfo, QuerySupplyResponse {}
 /** combines the ledger `BatchInfo` with ledger `QueryBalanceResponse` */
 export interface BatchInfoWithBalance extends BatchInfo, QueryBalanceResponse {}
 
+interface Coin {
+  denom: string;
+  amount: string;
+}
+
+interface BankQueryBalanceResponse {
+  balance?: Coin;
+}
+
```

Comment:
could you move these interfaces below the comment (lines 39-40):
```
// The following interfaces should be removed once we migrate
// the current queries to use regen-js instead of REST
```